travis-ci/managers/fuzzit.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+set -x
+set -u
+
+REPO_ROOT=${REPO_ROOT:-$(pwd)}
+
+sudo bash -c "echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial main restricted universe multiverse' >>/etc/apt/sources.list"
+sudo apt-get update -y
+sudo apt-get build-dep systemd -y
+sudo apt-get install -y ninja-build python3-pip python3-setuptools
+pip3 install meson
+
+cd $REPO_ROOT
+export PATH="$HOME/.local/bin/:$PATH"
+tools/oss-fuzz.sh
+
+export FUZZIT_API_KEY=7c1bd82fe0927ffe1b4bf1e2e86cc812b28dfe08a7080a7bf498e98715884a163402ee37ba95d4b1637247deffcea43e
+export FUZZIT_ADDITIONAL_FILES="./out/src/shared/libsystemd-shared-242.so"
+export FUZZIT_ARGS="--type sanity --branch ${TRAVIS_BRANCH} --revision ${TRAVIS_COMMIT}"
```

Comment:
Would it be possible to add TRAVIS_PULL_REQUEST somewhere to make it easier to figure out what's going on. Currently it seems to be hard to match jobs with PRs:
![Screenshot 2019-06-11 at 15 15 56](https://user-images.githubusercontent.com/10195800/59271141-e09fea80-8c5b-11e9-924a-eb670f481b7f.png)

Reply:
Sure, easy. will do today.